static/js/stream_muting.js

```diff
@@ -67,4 +68,20 @@ export function update_is_muted(sub, value) {
         )}'] #sub_is_muted_setting .sub_setting_control`,
     );
     is_muted_checkbox.prop("checked", value);
+
+    const row = $(`.stream-row[data-stream-id='${CSS.escape(sub.stream_id)}']`);
+    update_ui_muted_stream_checkboxes(row, value);
+}
+
+export function update_ui_muted_stream_checkboxes(row, value) {
+    $(row).toggleClass("control-label-disabled", value);
+    if (value) {
+        $(row).find(".unmute_stream").show();
+    } else {
+        $(row).find(".unmute_stream").hide();
+    }
+    $(row).find("input[disabled]").prop("disabled", value);
```

Comment:
This should be just `input` without `[disabled]` since we are using this code to both enable and disable the checkboxes as per whether the stream is muted or unmuted.